lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -71,7 +81,7 @@ class CategoryRenderer {
       const textEl = this.dom.find('.lh-audit__display-text', auditEl);
       textEl.textContent = 'Error!';
       textEl.classList.add('tooltip-boundary');
-      const tooltip = this.dom.createChildOf(textEl, 'div', 'lh-error-tooltip-content tooltip');
+      const tooltip = this.dom.createChildOf(textEl, 'div', 'tooltip lh-debug');
```

Comment:
does it make sense for the tooltip to share the `lh-debug` class?

Reply:
yes